platform/android/export/export_plugin.cpp

```diff
@@ -3290,4 +3290,6 @@ EditorExportPlatformAndroid::~EditorExportPlatformAndroid() {
 	quit_request.set();
 	check_for_changes_thread.wait_to_finish();
 #endif
+
+	env_script_encryption_key = "GODOT_ANDROID_SCRIPT_ENCRYPTION_KEY";
```

Comment:
Is this intentional?

Reply:
Shiiiiet. No, it is not

Reply:
If we had only one environment variable for the script encryption key, we'd introduce a lot of inconsistency and potential for confusion. Because we do have per-platform script encryption keys. The key is part of the individual presets, after all, so using the export UI already gives you a key per platform.

That's not even the whole truth, strictly speaking. You can have multiple presets for the same platform, in which case even per-platform environment variables are not enough and will start to break down. I just decided that that was too much of a niche case to spend time on it.

With all that said, I actually support having only one environment variable for the script encryption key, but I didn't want to make that decision on behalf of all users.

Reply:
Based on your comment, having per-platform script encryption keys seems to be more about addressing an implementation detail rather than server a user need. 
Based on that, I am leaning toward having a single environment variable for the script encryption key for all platform. This also serves the general use-case where we can assume that the user will use the same key to encrypt the project scripts in order to reduce logistics complexity (having to store / track / remember multiple keys for all supported platforms).